js/src/forum/components/Category.tsx

```diff
@@ -17,11 +17,18 @@ interface Attrs {
   parent: any;
 }
 
+interface TagLocked {
```

Comment:
I'm not completely opposed to including features for the tag lock extension here, but would it be possible to add a comment here explaining which extension is adding this?

Reply:
Added comment for reference of Flarum Tag Passwords